@app/db/scripts/toggle-admin.js

```diff
@@ -0,0 +1,28 @@
+const pg = require("pg");
+
+async function main() {
+  const pgPool = new pg.Pool({
+    host: process.env.DATABASE_HOST,
+    user: process.env.DATABASE_OWNER,
+    password: process.env.DATABASE_OWNER_PASSWORD,
+    database: process.env.DATABASE_NAME,
```

Comment:
Please use a connection string instead; it may also include details such as SSL certificate/etc in production. To do this you need to use `node -r @app/config/env scripts/toggle-admin.js` to call the script in the relevant `package.json` script.